index.js

```diff
@@ -288,7 +288,7 @@ function loadConfig(program) {
     }
 
     debug('connecting to log server')
-    connectionManager.connectLogServer(config)
+    setupLoggerThenStart(config)
```

Comment:
maybe InitializeLogger? Am thinking about naming for this function

Reply:
But this is the entrypoint for also starting the scanning of the logs, so that's important to have in the name.  It's also worth nothing that, although this is a small function, it's necessary because it helps testing.  It cuts out all of the cruft in index.js that we don't need or want for every test.

Reply:
If you like the word `initialize` better, I'm happy to change it.

Reply:
actually, I'll get consistent with this and call it `createLoggerThenStart` cause that's what it does :]. That is the closest to the actual function names too.